commands/events.js

```diff
@@ -88,7 +88,7 @@ function buildEmbeds(events, message) {
   message.reply('Here\'s the upcoming events:');
 
   for (let embed of embeds) {
-    message.channel.sendEmbed(embed);
+    message.channel.send(embed);
```

Comment:
This needs to be `.send({embed: embed});` otherwise Discord will complain about an empty message.